Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -104,6 +105,23 @@ private void SetGridView()
             grdExecutionsHistory.AddToolbarTool("@Delete_16x16.png", "Delete Selected Execution Results", new RoutedEventHandler(DeleteSelectedExecutionResults));
             grdExecutionsHistory.AddToolbarTool("@Trash_16x16.png", "Delete All Execution Results", new RoutedEventHandler(DeleteAllSelectedExecutionResults));
             grdExecutionsHistory.RowDoubleClick += OpenExecutionResultsFolder;
+
+            if (mExecutionHistoryLevel == eExecutionHistoryLevel.SpecificRunSet)
+            {
+                grdExecutionsHistory.AddCheckBox("Auto Load Execution History", new RoutedEventHandler(AutoLoadExecutionHistory));
+            }
+        }
+
+        private void AutoLoadExecutionHistory(object sender, RoutedEventArgs e)
+        {
```

Comment:
not critical for this case but always better to use binding for connecting between UI control and backend field